src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+      readOnly: true
+    - mountPath: "/app/certs"
+      name: orch-key
+      readOnly: true
+~~~
+
+Set the [corresponding config keys](#cert-var) to the same mount path. For example, within `config.json` in the [external secret](#manage-external-secret):
+
+~~~ yaml
+config.json: |
+  {
+    "CA_TLS_CERT": "/app/certs",
+    "TLS_CERT": "/app/certs",
+    "TLS_KEY": "/app/certs",
+  }
+~~~
+
+This also requires that you create and specify a CLI client certificate, key, and (optional) CA certificate. It's easiest to specify these with the following environment variables:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CA_CERT="{path-to-cli-ca-cert}"
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CLIENT_CERT="{path-to-cli-client-cert}"
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export CLI_TLS_CLIENT_KEY="{path-to-cli-client-key}"
+~~~
+
+## `molt-lms-cli`
+
+The `molt-lms-cli` command-line interface is used to inspect the LMS instances and [perform cutover](#perform-a-cutover).
+
+To install `molt-lms-cli`, [download the binary](https://molt.cockroachdb.com/).
+
+### Commands
+
+|       Command        |                                                                                     Usage                                                                                      |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `initialize`         | Set up the required objects for running the LMS. You must run this before using the LMS.                                                                                       |
+| `connections list`   | List all client connections to the LMS and their most recent queries.                                                                                                          |
+| `cutover consistent` | Specify a [consistent cutover](#consistent-cutover). You must also specify `begin`, `commit`, or `abort`. For usage details, see [Consistent cutover](#consistent-cutover).    |
+| `begin`              | Begin a consistent cutover. This pauses traffic to the source of truth.                                                                                                        |
+| `commit`             | Commit a consistent cutover. This switches the source of truth to the target database.                                                                                         |
+| `abort`              | Abort a consistent cutover after running `consistent cutover begin`, unless you have also run `consistent cutover commit`. This resumes traffic to the source of truth.        |
+| `cutover immediate`  | Initiate an [immediate cutover](#immediate-cutover). This switches the source of truth to the target database. For usage details, see [Immediate cutover](#immediate-cutover). |
```

Comment:
@cotedm @ZhouXing19 @taroface this is currently a `TODO` command. Should we remove this from both the binary and the docs?

Reply:
Yes, if it is a `TODO` let's remove it. @taroface this will also mean hiding the section on Immediate cutover for now until we have this in the CLI.

Reply:
Will this be a `TODO` until Roachfest?

Reply:
Yes immediate cutover won't be ready on Roachfest.